torch_xla/core/dynamo_bridge.py

```diff
@@ -147,14 +146,19 @@ def _maybe_move_tensors_to_device(tensors: tuple,
       moved_tensors.append(tensor)
       continue
 
-    assert target_device is not None, "Moving tensors to None device not supported"
-
     if dynamo_debug:
       print("Moving Tensor {} to device {}".format(tensor, target_device))
 
-    # Have to move to CPU before moving it to target device.
-    moved_tensor = tensor.to(cpu_device)
-    moved_tensor = moved_tensor.to(target_device)
+    zero_copy_enabled = xu.getenv_as(xenv.ZERO_COPY_ENABLED, bool, defval=False)
```

Comment:
is there a reason this has to be a env var? Biggest reason we use env var is we need to communcate something between python and C++ layers, so it is the master process need to pass some information(like rank) to the child process. In your case this is really just a config somewhere and should not be set as env var.

Reply:
for you case I think you can just always use dlpack, is there any reason we don't want to use dlpack to convert XLA:GPU and cuad?

Reply:
The flag is only temporary. It's use to do a/b test: how much performance when we move tensor through CPU vs how much performance we can get by using dlpack.

So the temporary  be removed later once the a/b testing is done.